spec/granite/form/model/attributes/represents_spec.rb

```diff
@@ -105,110 +105,4 @@ def typecast(value)
       expect(typecast(nil)).to be_nil
     end
   end
-
-  describe '#type_definition' do
```

Comment:
Moved with slight modifications to `spec/granite/form/model/attributes/reflections/represents/build_type_definition_spec.rb`